xml/System.Diagnostics/Activity.xml

```diff
@@ -352,9 +353,17 @@ In .NET 5.0 and later, the default format is <xref:System.Diagnostics.ActivityId
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the display name of the activity.</summary>
+        <value>A string that represents the display name of the activity.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+`DisplayName` is intended to be used in a user interface and need not be the same as `OperationName`.
```

Comment:
>DisplayName [](start = 1, length = 11)

can you add a sentence here saying, if the DisplayName is not set by the user, the getter of DisplayName will return the value of Activity.Name. 

feel free to rephrase it.